src/main/java/com/github/manolo8/darkbot/core/objects/facades/AstralSelectionProxy.java

```diff
@@ -0,0 +1,10 @@
+package com.github.manolo8.darkbot.core.objects.facades;
+
+import com.github.manolo8.darkbot.core.itf.Updatable;
+
+public class AstralSelectionProxy extends Updatable {
+    @Override
+    public void update() {
+
+    }
```

Comment:
A proxy with no code isn't terribly useful, i'd just delete this class unless it's s till WIP and you plan on finishing it within this PR